src/app/header/header.component.html

```diff
@@ -1,28 +1,21 @@
 <nav class="navbar navbar-default">
   <div class="container-fluid">
     <div class="navbar-header">
-      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false">
+      <button type="button" (click)="isShown=!isShown" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false">
```

Comment:
please remove attributes that you do not need , what is data-toggle , data-target and aria-expand are for?

Reply:
those are bootstrap classes

Reply:
i don't take about the classes. there are more attributes there. please remove once that sre not needed. try to remove data-taget and data-expanded and see if it still works.